orbit/pkg/setup_experience/setup_experience.go

```diff
@@ -144,17 +146,24 @@ func (s *SetupExperiencer) Run(oc *fleet.OrbitConfig) error {
 		}
 
 		for _, step := range steps {
-			item := resultToListItem(step)
-			if _, ok := s.steps[step.Name]; ok {
-				err = s.sd.UpdateListItemByTitle(item.Title, item.StatusText, item.Status)
-				if err != nil {
-					log.Info().Err(err).Msg("updating list item in setup experience UI")
+			currentStepState := resultToListItem(step)
+			if priorStepState, ok := s.uiSteps[step.Name]; ok {
+				if currentStepState != priorStepState {
+					// We only want to resend on change so we're not unnecessarily scrolling the UI
+					err = s.sd.UpdateListItemByTitle(currentStepState.Title, currentStepState.StatusText, currentStepState.Status)
+					if err != nil {
+						log.Info().Err(err).Msg("updating list item in setup experience UI")
+					}
+				} else {
+					log.Debug().Msgf("setup experience: no change in status for %s, skipping update", step.Name)
```

Comment:
Let's make this an info level log to give us better visibility into the setup experience in general.

```suggestion
					log.Info().Msgf("setup experience: no change in status for %s", step.Name)
```

Reply:
do we care if this spams the logs at all? I guess it's only during setup experience so probably not a big deal but coudl result in a deecnt bit of logging on each setup experience ping if they have a lot of items to install

Reply:
For setup experience, I think it is ok to err on the side of over-logging. In my experience trying to debug setup experience issues, I always find myself wishing that we had more logs, especially because the issues often don't repro consistently in our dev environments and there's no good process to change the fleetd log level in production, especially for new devices are going through setup.

Reply:
Or maybe it would be better to make it just one line if you want to append to a message string as you loop each step and just log it when you exit the loop.